app/data/generators/events.js

```diff
@@ -1,9 +1,17 @@
+const { DateTime } = require('luxon')
+
+function getDate(faker, date = new Date, min = 1, max = 20) {
+  let dt = DateTime.fromJSDate(date)
+  const num = faker.random.number({ 'min': min, 'max': max })
+  return dt.plus({ days: num }).toJSDate()
+}
+
 module.exports = (faker, params) => {
   const events = { items: [] }
 
-  const date = faker.helpers.randomize([
-    '2020-08-12',
```

Comment:
The reason this was hard coded is because the prototype now date is set to: 2020-08-15.